lib/apis/v3/bgpconfig.go

```diff
@@ -55,6 +55,15 @@ type BGPConfigurationSpec struct {
 	// ServiceClusterIPs are the CIDR blocks from which service cluster IPs are allocated.
 	// If specified, Calico will advertise these blocks, as well as any cluster IPs within them.
 	ServiceClusterIPs []ServiceClusterIPBlock `json:"serviceClusterIPs,omitempty" validate:"omitempty,dive" confignamev1:"svc_cluster_ips"`
+
+	// Communities contain list of BGP community values and their arbitrary names for tagging routes.
+	Communities []Community `json:"communities,omitempty" validate:"omitempty,dive" confignamev1:"communities"`
+
+	// PrefixAdvertisement lists the communities to be advertised per prefix.
```

Comment:
This comment (while technically correct for now) isn't how I think we should document this field.

It should be something like this:

```
PrefixAdvertisements contains per-prefix advertisement configuration.
```

Reply:
The distinction being that while Communities are the only thing you can currently set here, the PrefixAdvertisements sub-section is intended to be more general.

Also, this comment has a typo: "PrefixAdvertisement" -> "PrefixAdvertisements"